### PR TITLE
testing/audit: initial import

### PR DESCRIPTION
**Pull request by tych0**
An initial import of the audit daemon, libraries, and utilities.

Signed-off-by: Tycho Andersen <tycho@docker.com>

### CODE REVIEW

**Comment by kaniini** [comment]
The package builds fine.

With that said, can you clarify on your (or perhaps Docker's) overall goals / plans with integrating audit support into the distribution?  Such information would be useful in evaluating whether or not to include this.

File: testing/audit/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+    --disable-zos-remote \
```
Comment (kaniini) [inline testing/audit/APKBUILD]: indentation

File: testing/audit/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+depends=""
+depends_dev=""
+makedepends="$depends_dev linux-headers libcap-ng-dev python3"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="http://people.redhat.com/sgrubb/audit/audit-$pkgver.tar.gz
+        0001-auditctl-include-headers-to-make-build-work-with-mus.patch
+        0002-auparse-remove-use-of-rawmemchr.patch
+        0003-all-get-rid-of-strndupa.patch
+        "
```
Comment (kaniini) [inline testing/audit/APKBUILD]: auditd should have an init script

**Comment by kaniini** [comment]
I went ahead and put this in testing for now, but I think in order to really deliver the most potential we should add an init script and move it into main.

It would be appreciated if you could work on the init script and indentation cleanups.

**Comment by tych0** [comment]
Hi,

Just for posterity: our goal is to get the selinux userspace built for alpine, and it depends on libaudit (and auditd) for logging things, IIUC. I should have some more PRs coming soon for SELinux libraries and such.

Thanks for the merge! I cleaned up the indentation in a new PR, and I'll see about doing the init script later this week.

**Pull request closed without merging**